src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -1127,7 +1127,10 @@ public override Task<ImplementationResult> ImplementationAsync(DelegatedPosition
         }
         catch (Exception e)
         {
-            _outputWindowLogger?.LogError(e, "Exception thrown in PullDiagnostic delegation");
+            if (e is not OperationCanceledException)
```

Comment:
This luckily also gets TaskCancelledException, which I assume we wouldn't want to log either